tsdb/mapper_test.go

```diff
@@ -136,41 +136,41 @@ func TestShardMapper_WriteAndSingleMapperRawQuery(t *testing.T) {
 	}{
 		{
 			stmt:     `SELECT load FROM cpu`,
-			expected: []string{`{"name":"cpu","fields":["load"],"values":[{"time":1000000000,"value":42},{"time":2000000000,"value":60}]}`, `null`},
+			expected: []string{`{"name":"cpu","fields":["load"],"values":[{"time":1000000000,"value":42,"tags":{"host":"serverA","region":"us-east"}},{"time":2000000000,"value":60,"tags":{"host":"serverB","region":"us-east"}}]}`, `null`},
 		},
 		{
-			stmt:      `SELECT load FROM cpu`,
+			stmt:      `SELECT load FROM cpu # chunkSize 1`,
```

Comment:
I thought so :-).  The issue I had was that I couldn't tell what test was actually failing because the statements were identical.  This worked out nicely.